client/ayon_core/hosts/zbrush/api/pipeline.py

```diff
@@ -147,12 +147,20 @@ def list_instances(self):
         return get_instance_workfile_metadata(ZBRUSH_SECTION_NAME_INSTANCES)
 
     def write_instances(self, data):
+        """Write all AYON instances"""
         return write_workfile_metadata(ZBRUSH_SECTION_NAME_INSTANCES, data)
 
     def get_containers(self):
+        """Get the data of the containers
+
+        Returns:
+            list: the list which stores the data of the containers
+        """
         return get_containers()
 
     def initial_app_launch(self):
+        """The actions should be done when Zbrush launches.
+        """
```

Comment:
```suggestion
        """Triggers on launch of the communication server for Zbrush.
        
        Usually this aligns roughly with the start of Zbrush.
        """
```
I wonder if this _may_ be before Zbrush actually fully launched. If so, we may want to include that in the docstring